### PR TITLE
Fix deny all authorization policy examples

### PR DESCRIPTION
**Pull request by rolandkool**
Fix for https://github.com/istio/api/issues/1183

### CODE REVIEW

**Comment by geeknoid** [comment]
I suspect you'll get a CI failure with the gencheck_api job. To fix this, run "make gen" and then add any changed files to the PR.

Thanks.

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Comment by geeknoid** [comment]
Hi again, it appears our CLA bot is still not satisfied. It's looking for a CLA for the email address rkool@bol.com. Is it possible you signed the CLA using a different address?

Sorry for all the hoops just to submit this simple change. The change is appreciated, even if our legal bot is annoying...

**Comment by rolandkool** [comment]
Hi @geeknoid 
Not sure how it got my work email address, I didn't sign the CLA myself, but someone from my work did. We set up a Google group, and I'm part of that group. I also added the email address to that group that I used for this PR. What options are left to make this CLA bot happy?

**Comment by howardjohn** [comment]
@rolandkool I think when you sign an individual CLA its auto approved, but corporate ones need to be manually approved. Last time I saw this it took 1-2 days, but it may have been delayed by holidays?

**Comment by rolandkool** [comment]
I know that our Corporate CLA was signed / approved but I can double check to make sure nothing was missed. As mentioned before, we've setup a separate gsuite group as suggested in the corporate CLA form. I'm part of that group with my corporate email address (I realize now that that email address is used for committing changes and not my private one, which is assigned to my GH account).
Feel free to reach out to me directly to help me figure out what is wrong.

**Comment by howardjohn** [comment]
@googlebot

**Comment by howardjohn** [comment]
@rolandkool I finally found the page of corporate CLAs signed, and your company. On the page it says "if it's not, it simply may not have been processed yet, which typically takes a few days." So if it doesn't show up in the next few days I'll reach out to the team and see whats going on. Sorry!

**Comment by rolandkool** [comment]
Al right, so it's waiting approval on the Google side as well. I'll call the googlebot later this week.

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Comment by louiscryan** [comment]
@yangminzhu @liminw please review

File: security/v1beta1/authorization.proto [hunk 1]
```diff
@@ -139,7 +139,7 @@ option go_package="istio.io/api/security/v1beta1";
 // metadata:
 //  name: deny-all
 //  namespace: foo
-// spec:
```
Comment (yangminzhu) [inline security/v1beta1/authorization.proto]: Could you put the `{}` in next line to align with example we used in other places? e.g. https://istio.io/docs/concepts/security/#allow-all-and-deny-all,  https://istio.io/docs/tasks/security/authorization/authz-http/#configure-access-control-for-workloads-using-http-traffic and etc.

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Comment by rolandkool** [comment]
@googlebot I signed it!

**Pull request merged**